packages/apps-config/src/ui/colors.ts

```diff
@@ -236,8 +238,6 @@ export const chainColors: Record<string, string> = Object.entries({
   PHOENIX: chainPhoenix,
   Plasm: chainPlasm,
   'Plasm PC2': chainPlasm,
-  PolkaBTC: chainPolkabtc,
```

Comment:
This is possibly the same - just needs to be kept for the time being.

Reply:
The problem was that there are two versions of the interbtc-parachain: one with `aura-grandpa` consensus (for testing) and one with `cumulus`. I included both in `api/spec/index.ts` and it works now - I think it's fine to remove the PolkaBTC configs

Reply:
All ok. The above is just for the color matching.